src/org/usfirst/frc/team166/robot/RobotMap.java

```diff
@@ -12,6 +12,7 @@
 	public static int reverseSolenoid = 0;
 	// Motor Controllers
 	public static int frontRightMotor = 1;
+	public static int elevatorMotor = 2;
```

Comment:
Please move this so it's not interspersed with the drive motors.

The four drive motors are CAN, so the `elevatorMotor` being a PWM value won't conflict.